lib/ace/ext/papeeria/spellchecker_popup.coffee

```diff
@@ -16,7 +16,7 @@ define([
   # @param {Array} corrections: array of strings with substitution options.
   # @return {Array}: array of JSONs, actually.
   convertCorrectionList = (corrections) ->
-    return ({caption: item, value: item} for item in corrections)
+    return ({caption: item, value: item, meta: item, meta_score: corrections.length - i} for item, i in corrections)
```

Comment:
I think `meta` should indicate the source of correction: is it built-in dictionary, private dictionary or shared dictionary (if we come up with such features). So maybe modify JSON structure so that we had sources, or add TODO to make in the future, and remove this `meta` now?

`meta_score` applies to meta, as to the best of my knowledge. Did you mean to use just `score`?